addons/website_sale/views/templates.xml

```diff
@@ -480,8 +480,8 @@
                             <p t-elif="not product.active" class="alert alert-warning">This product is no longer available.</p>
                             <p t-else="" class="alert alert-warning">This product has no valid combination.</p>
                             <hr t-if="product.description_sale" />
-                            <div class="o_not_editable">
-                                <p t-field="product.description_sale" class="text-muted" />
+                            <div>
+                                <p t-field="product.description_sale" class="text-muted mt-3" placeholder="Type here the description of your product."/>
```

Comment:
Please change the placeholder to
> A short description that will also appear on documents.

As this field primary purpose is to be shown on SO/invoices we want to make sure it is clear that user should not write a big description here and understand it will have impact on backend

Reply:
done.!